c_binding/include/libstoragemgmt/libstoragemgmt_types.h

```diff
@@ -367,6 +372,35 @@ typedef enum {
 #define LSM_SYSTEM_CACHE_PCT_NO_SUPPORT             -2
 #define LSM_SYSTEM_CACHE_PCT_UNKNOWN                -1
 
+typedef enum {
+    LSM_BATTERY_TYPE_UNKNOWN = 1,
+    LSM_BATTERY_TYPE_OTHER = 2,
+    LSM_BATTERY_TYPE_CHEMICAL = 3,
+    LSM_BATTERY_TYPE_CAPACITOR = 4,
+} lsm_battery_type;
+
+/** Unknown. */
+#define LSM_BATTERY_STATUS_UNKNOWN                  1 << 0
+/** Vendor specific status. */
+#define LSM_BATTERY_STATUS_OTHER                    1 << 1
+/** Battery is fully charged, health and not in use currently. */
+#define LSM_BATTERY_STATUS_OK                       1 << 2
+/** Battery is in use. */
+#define LSM_BATTERY_STATUS_DISCHARGING              1 << 3
+/** Battery is charging. */
+#define LSM_BATTERY_STATUS_CHARGING                 1 << 4
+/** Battery is calibrating itself by discharging battery and
+ *  recharging again.
+ */
+#define LSM_BATTERY_STATUS_LEARNING                 1 << 5
+/** Battery is in degraded mode, needs attention.
+ *  For example, battery is near end of life.
+ */
+#define LSM_BATTERY_STATUS_DEGRADED                 1 << 6
+/** Battery is having hardware error or end of life. */
+#define LSM_BATTERY_STATUS_ERROR                    1 << 7
+
```

Comment:
All the other `#define` for bitfield values are declared using `0x0000000000000001`, `0x0000000000000002`, please make consistent.